webofneeds/won-matcher-siren/pom.xml

```diff
@@ -52,6 +52,16 @@
             <artifactId>jena-arq</artifactId>
         </dependency>
         <dependency>
+            <groupId>org.apache.commons</groupId>
+            <artifactId>commons-math3</artifactId>
+            <version>3.6.1</version>
```

Comment:
please add the dependency to the `<dependencyManagement>` element of the parent pom and remove the version info here